RELEASE.md

```diff
@@ -1,3 +1,12 @@
+# Release 0.17.9
+
+## Major features and improvements
+* Added abfss to list of cloud protocols, enabling abfss paths.
```

Comment:
We don't have a 0.17.8 release either, could we put these below one section please? :)